docs/user_guide/assessment/common_fairness_metrics.rst

```diff
@@ -299,7 +305,128 @@ capture the costs of missclassification disparities.
+terminology in our documentation.
+
+For a much deeper discussion of the issues involved, we suggest
+:footcite:ct:`watkins2022fourfifths`.
+A higher level look at how legal concepts of fairness can collide
+with mathematical measures of disparity, see
+:footcite:ct:`Xiang2019legalcompatibility`.
+
+
+Summary
+-------
+
+We have introduced three commonly used fairness metrics in this section,
+which can be summed up as follows:
+
+* Demographic Parity
+
+    * *What it compares:* Predictions between different groups
+      (true values are ignored)
+
+    * *Reason to use:* If the input data are known to contain
+      biases, demographic parity may be appropriate to measure fairness
+
+    * *Caveats:* By only using the predicted values, information is thrown
+      away. The selection rate is also a very coarse measure of the
+      distribution between groups, making it tricky to use as an optimization
+      constraint
+
+* Equalized Odds
+
+    * *What it compares:* True and False Positive rates between different groups
+
+    * *Reason to use:* If historical data does not contain measurement bias or historical
+      bias that we need to take into account, and true and false positives
+      are considered to be (roughly) of the same importance, equalized odds may be useful
+
+    * *Caveats:* If there are historical biases in the data, then the original labels
+      may hold little value. A large imbalance between the positive and negative classes
+      will also accentuate any statistical issues related to sensitive groups with low
+      membership
+
+* Equal opportunity
+
+    * *What it compares:* True Positive rates between different groups
+
+    * *Reason to use:* If historical data are useful, and extra false positives
+      are much less likely to cause harm than missed true positives, equal
+      opportunity may be useful
+
+    * *Caveats:* If there are historical biases in the data, then the original labels
+      may hold little value. A large imbalance between the positive and negative classes
+      will also accentuate any statistical issues related to sensitive groups with low
+      membership
+
+
+However, the fact these are common metrics does not make them applicable to any given
+situation.
+In particular, :ref:`demographic parity is often misapplied <assessment_four_fifths>`.
+
+
 References
 ----------
 
-.. footbibliography::
\ No newline at end of file
+.. footbibliography::
+
+
+.. rubric:: Footnotes
+
+.. [#f1] For a related example, see the discussion on 'law' and 'justice' in
+         *The Caves of Steel* (Asimov, 1953)
+
+.. [#f2] Epistemic trespassing is the process of taking expertise in one field and
+         applying it to another. This is not an intrinsically bad thing - one could
+         label all interdisciplinary research a form of epistemic trespassing.
+         However, doing so successfully requires a willingness to learn the subtleties
+         of the new field.
```

Comment:
It's my understanding that epistemic trespassing involves some aspect of applying one's expertise in their own field to another *in which they do not have equivalent (or any) competence*, and that therefore the expertise or judgement may be incorrectly applied.  I just suggest making it a little clearer why this might be a problematic thing in the context of fairness assessment.

Reply:
Tweaked.